meson.build

```diff
@@ -43,7 +43,7 @@ else
   os = option_os
 endif
 
-if os != 'Linux'
+if os != 'Linux' and os != 'GNU'
```

Comment:
not a new issue, but this check seems to assume that we always want kvm on non-Linux systems.  this should really be changed to only require kvm on BSD systems